Documentation/Books/Manual/Views/ArangoSearch.md

```diff
@@ -201,33 +201,33 @@ During view modification the following directives apply:
 
     * bytes: (optional; for default values use an empty object: `{}`)
 
-      * intervalStep: (optional, default: `10`; to disable use: `0`)
-        apply consolidation policy with every Nth commit
+      * segmentThreshold: (optional, default: `300`; to disable use: `0`)
+        apply consolidation policy IFF {segmentThreshold} >= #segments
 
       * threshold: (optional; default: `0.85`)
         consolidate `IFF {threshold} > segment_bytes / (all_segment_bytes / #segments)`
```

Comment:
why IFF here? shouldn't it rather be IF ?

Reply:
IFF is the common shorthand for "if and only if"

Reply:
hm, would it make sense to add a link to https://en.m.wikipedia.org/wiki/If_and_only_if in the Glossary then? https://docs.arangodb.com/3.3/Manual/Appendix/Glossary.html

Reply:
IFF is quite often used in tech or scientific environments.I think it's not too uncommon to use it in rather technical documentation.